spharpy/samplings/coordinates.py

```diff
@@ -303,3 +354,36 @@ def n_max(self, value):
             self._n_max = None
         else:
             self._n_max = int(value)
+
+    @property
+    def radius_tolerance(self):
+        """Get or set the radius tolerance in meter."""
+        return self._radius_tolerance
+
+    @radius_tolerance.setter
+    def radius_tolerance(self, value):
+        """Get or set the radius tolerance in meter."""
+        if isinstance(value, (int, float)) and value > 0:
```

Comment:
is we change the setter on runtime, we need to check the radius again.
maybe its better to set the radius at the beginning and dont allow to change it anymore. otherwise we could call 

self._x,  self._y,  self._z = self._check_points(self._x,  self._y,  self._z)